### DISCUSSION TITLE
ProfileSystemMetadata

### DISCUSSION THREAD

**Discussion by NameArtem** [Tracardi/tracardi#853] [Q&A]
Hello!
I stack with problem after running containers.

Errors in tracardi-api container "ERROR:tracardi.service.tracker:'ProfileSystemMetadata' object has no attribute 'metadata'"
The same in a front page:
![image](https://github.com/Tracardi/tracardi/assets/18335444/5509018b-553d-4cfe-ba4c-5a2844580462)

How can I solve it?

**Comment by madanvparadkar** [comment]
I too was facing a similar issue. I stopped docker service and started it again. It worked for me.
